packages/cli/README.md

```diff
@@ -147,7 +159,7 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
 ```
 
 ### `glaze did:set MODEL ALIAS CONTENTS`
```

Comment:
text description of this command says "get" the contents when it should probably be "set" the content